<!DOCTYPE html >
<html id="aZOkHe0AzLPKEKUodoDFwMBBrlxO-_10.1111_bij.12796.pdf" data-origid="[10.1111]bij.12796.pdf" class="anndoc" data-anndoc-version="3.6" lang="" xml:lang="" xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="UTF-8"/>
    <meta name="generator" content="net.tagtog.anndoc.v3.parsers.general.pdf.v2.NativePdfParser_v2_0_0"/>
    <title>aZOkHe0AzLPKEKUodoDFwMBBrlxO-_10.1111_bij.12796.pdf</title>
  </head>
  <body>
    <article>
      <section data-type="">
        <div class="content">
          <pre id="s1v1">Original Article.1

High levels of intra-specific genetic divergences revealed for Antarctic2

springtails: evidence for small-scale isolation during Pleistocene glaciation3

Kristi R. Bennett1, Ian D. Hogg1, Byron J. Adams2, Paul D.N. Hebert34

1 School of Science, University of Waikato, Private Bag 3105, Hamilton 3240, New Zealand5

2 Department of Biology and Evolutionary Ecology Laboratories, Brigham Young University,6

4102 LSB, Provo, UT 84602-5253, USA7

3 Biodiversity Institute of Ontario, University of Guelph, 50 Stone Road East, Guelph, ON8

N1G 2W1, Canada9

Running Head: Genetic divergences among Antarctic Springtails10

Correspondence:11

Ian Hogg12

School of Science 13

University of Waikato 14

Private Bag 3105 15

Hamilton 3240, New Zealand16 email: hogg@waikato.ac.nz 17

Abstract word count: 233.  Whole text word count: 6,631.18</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s2v1">Abstract19

Aim To examine the levels of genetic variability within and among populations of three20

Antarctic springtail species (Arthropoda: Collembola) and test the hypothesis that genetic21 divergences occur among glacially-isolated habitats.22

Location Southern Victoria Land, Ross Dependency, Antarctica.23

Methods Samples were collected from locations in the vicinity of the Mackay Glacier. We24 analysed mitochondrial DNA (COI) sequence variability for 97 individuals representing three25 species (Gomphiocephalus hodgsoni n=67; Cryptopygus nivicolus, n=20; Antarcticinella26 monoculata, n=8). Haplotype diversity and genetic divergences were calculated and used to27 indicate population variability as well as infer divergence times of isolated populations using28 molecular clock estimates. 29

Results Two of the three species showed high levels of genetic divergence.30

Gomphiocephalus hodgsoni, a widespread and common species showed 7.6% sequence31 divergence on opposite sides of the Mackay Glacier. The more range restricted Cryptopygus32 nivicolus species showed 4.0% divergence among populations. The third species,33

Antarcticinella monoculata, was found in only one location. Molecular clock estimates based34 on sequence divergences suggest that populations separated within the last 4 Mya.35

Main Conclusions Habitat fragmentation resulting from Pliocene (5 Mya) and Pleistocene (236

Mya - 10 Kya) glaciations has promoted and maintained high levels of diversity among37 isolated springtail populations on relatively small spatial scales. The region surrounding the38

Mackay Glacier has provided refugia for springtail populations during glacial maxima and39 remains an area of high genetic and species diversity for Collembola within the Ross Sea40 region.41</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s3v1">Key Words Collembola, glaciation, Ross Sea region, population genetics, springtails,42 refugia.43



44



45



46



47



48



49



50



51



52



53



54



55



56



57



58



59</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s4v1">Introduction60

With only 0.34% (46,200 km²) of the total 14 million km² ice free and even marginally61 habitable, the Antarctic continent represents one of the most extreme environments for62 terrestrial life (Convey et al., 2009; Hogg &amp; Wall, 2012). The majority of these ice-free areas63 lie within the Dry Valleys and Transantarctic Mountains of the Ross Dependency64

(Janetschek, 1967a; Levy, 2012). Even here, exposed ground is often highly fragmented and65 comprised of small, rocky outcrops separated by permanent snow fields and glaciers. Suitable66 habitat is then further restricted by the availability of liquid water necessary to support life67

(Hogg et al., 2006). This latter requirement is relevant for the soil arthropod fauna,68 particularly the Antarctic springtails which lack a desiccation-resistant life stage and instead69 use avoidance and super-cooling methods to allow survival in sub-zero temperatures70

(McGaughran et al., 2011a).71

The terrestrial arthropods are represented primarily by springtails (Collembola) and mites72

(Acari) and are the largest year-round taxa on the continent (Gressitt, 1967; Hogg &amp; Stevens,73

2002; Adams et al., 2006). These taxa, which lack survival and dispersal strategies possessed74 by other invertebrate groups such as nematodes (Adhikari et al.,2010; Nkem et al., 2006),75 have been restricted to these fragmented, ice-free zones since the Middle Miocene (14-1176

Mya; Stevens &amp; Hogg, 2003; Stevens et al., 2006; McGaughran et al., 2010). At this time,77 glaciation of the whole continent reached its fullest extent and the polar ice cap overflowed78 the Transantarctic Mountains (Lewis et al., 2007) . Small oases of ice-free ground existed79 around the edge of the polar cap, the largest of which (the Dry Valleys) is still located within80 the Transantarctic Mountain on the western edge of the Ross Ice Shelf (Clapperton &amp;81

Sugden, 1990). Since then, the East Antarctic Ice Sheet (EAIS) has undergone numerous82 glacial cycles, with the last glacial maximum ending 17 Kya (Suggate, 1990). This extensive83</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s5v1">glacial history has resulted in extremely low species richness for the Antarctic fauna, with84 many habitats containing at most one or two arthropod taxa (Janetschek, 1967a). Species are85 also rarely shared between regions (Gressitt, 1967; Wise, 1971; Sinclair &amp; Stevens, 2006),86 suggesting limited inter-habitat dispersal. Consequently, the current arthropod taxa are likely87 to be long-term inhabitants and remnants of, once more widespread species (Convey et al.,88

2009). Even within regions, most species show high levels of genetic divergence across their89 distributional ranges suggesting the effects of long-term isolation and/or survival in glacial90 refugia (Frati et al., 2001; Stevens &amp; Hogg, 2003; McGaughran et al., 2008; Hawes et al.,91

2010; Stevens &amp; D'Haese, 2014). Here, our aim was to extend these studies by focussing on92 small-scale differences that might occur within faunally-diverse, yet heavily fragmented,93 landscapes. 94

Ten species of springtail are currently known from the Ross Dependency, four in northern95

Victoria Land, three in southern Victoria Land and three in the southern Transantarctic96

Mountains. All species are range-restricted. Species from southern Victoria Land, the focus97 of our study, consist of three species covering a 3o latitudinal range. Within this region98

Gomphiocephalus hodgsoni is the only relatively widespread species and is common99 throughout southern Victoria Land (McGaughran et al., 2011b). Two additional species,100

Cryptopygus nivicolus (recently redescribed from Neocryptopygus nivicous by Greenslade,101

(2015)) and Antarcticinella monoculata are extremely range-restricted and known only from102 one or two locations near the Mackay Glacier to the north of the Dry Valleys (Wise, 1971)103

(Fig. 1), suggesting the possibility of a glacial refugium. Recent studies of lichens and mosses104 also near the Mackay Glacier (Green et al., 2011), as well as haplotype diversity for springtail105

(G. hodgsoni) and mite (Stereotydeus. mollis) taxa have further suggested this area as a likely106 refugial zone (Stevens &amp; Hogg, 2003, 2006; McGaughran et al., 2008; Demetras et al.,107

2010). 108</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s6v1">In order to determine the geographic scales on which genetic diversity may have been109 promoted and/or maintained, we focused on small-scale genetic variability in a region of110 comparatively high species diversity (Mackay Glacier, southern Victoria Land). This glacier111 is one of only a few outlet glaciers that connect the EAIS with the Ross Ice Shelf in southern112

Victoria Land (Clapperton &amp; Sugden, 1990). Accordingly, we tested the hypothesis that this113 region would support genetically divergent springtail populations among isolated habitats.114

We predicted that high levels of both genetic variability and genetic divergence would exist115 among these habitats, potentially indicating refugial zones from the Pleistocene glaciations.116

Methods117

Study sites and sample collection: 118

Samples were collected from St John's Ranges near Victoria Valley and on the northern and119 southern sides of the Mackay Glacier in the northern Dry Valleys region of the Ross120

Dependency (Fig. 1). Specimens were collected from the undersides of rocks using modified121 aspirators (Stevens &amp; Hogg, 2002). Soil samples were also taken from each site and returned122 to the lab where they were suspended a 10% sucrose solution. Invertebrates were then123 removed from the solution surface under a dissecting microscope (10X magnification) using124 a fine wire loop. All specimens were stored in 95% ethanol and returned to the University of125

Waikato for further processing. All specimens were morphologically identified to species126 level using Gressitt et al., (1963) and Salmon, (1965). Specimens not used for DNA analyses127 were archived at the School of Science, University of Waikato, under the care of IDH.128

Genetic analyses: 129

Genetic analyses were jointly carried out at the University of Waikato and at the Canadian130

Centre for DNA Barcoding (CCDB) at the University of Guelph. At the University of131</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s7v1">Waikato total genomic DNA was extracted from the tissue of entire specimens using a132

Glassfiber Plate DNA Extraction (AcroPrep) method (Ivanova et al., 2006) at CCDB, and133

Red Extract n Amp (Sigma-Aldrich) using 10 µl extraction solution and 2.5 µl tissue prep,134 following manufacture’s protocol. Polymerase Chain Reactions (PCRs) were comprised of a135

15 µl reaction containing 5.7 µl MQH20, 7.5 µl PCR Master Mix Solution (i-Taq, Intron136

Biotechnology), 0.4 µl of each primer and 1 µl of template DNA. A 658 bp fragment of the137 mitochondrial COI gene was amplified using the primers HCO2198 (sequence 5'-138

TAAACTTCAGGGTGACCAAAAAATCA-3') and an altered LCO1490 (sequence: 5'-139

AGTTCTAATCATTAARGATATYGG-3') (Folmer et al., 1994) for the G. hodgsoni140 specimens. HCO and LepF1 (sequence: 5'-ATTCAACCAATCATAAAGATATTGG-3')141

(Hajibabaei et al., 2006) were used to amplify the C. nivicolus and A. monoculata specimens.142

The standard LCO1490 (sequence: 5'-GGTCAACAAATCATAAAGATATTGG-3') was143 used for both species (in place of the altered LCO1490 and LepF1) at CCDB. Primers were144 used at 1.0 mM concentration. PCR conditions at CCDB were: initial denaturing at 94ºC for145

1 min; 5 cycles of 94ºC for 1 min, 45ºC for 1.5 min and 72ºC for 1.5 min; 35 cycles of 94ºC146 for 1 min, 50ºC for 1.5 min and 72ºC for 1 min followed by a final 72ºC for 5 min. PCR147 conditions were: initial denaturing at 94ºC for 5 minutes; 36 cycles of 94ºC for 1 min, 52ºC148 for 1.5 min and 72ºC for 1 min, followed by a final 72ºC for 5 min.149

PCR products were cleaned using Sephadex (CCDB) or 0.2 µl ExonucleaseI (EXO) and 0.1150 µl Shrimp Alkaline Phosphate (SAP) with 2.7 µl MQH20 following manufactures protocol151

(Global Science &amp; Tech Ltd.) at Waikato. DNA was sequenced in both directions on an152

ABI3130 sequencer at the University of Waikato DNA sequencing facility using the same153 primers used for amplification, or on an ABI3730x1 at CCDB. Sequences from the154

University of Waikato were aligned using Geneious, ver 5.4.2, and confirmed as the target155 species using the Barcode of Life DataSystems (BOLD; www.boldsystems.org) ver 3 COI156</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s8v1">animal identification searches. Previous analyses of Antarctic springtails (e.g. Stevens &amp;157

Hogg 2003), have shown that allozyme analyses were congruent with COI data and that the158 latter can be used as a reliable indicator of genomic differences occurring among populations.159

Primer sequences were trimmed from sequence fragments for further analyses. All sequences160 were uploaded to the BOLD project Antarctic Terrestrial Arthropods (ANTSP) and cross-161 referenced to GenBank. 162

Phylogenetic Analysis163

COI sequence fragments of 658 bp (219 codons) were obtained for 67 G. hodgsoni specimens164 and 20 C. nivicolus specimens. Approximately 560 bp were obtained from single direction165 reads (using primer LepF1) for eight A. monoculata specimens. No stop codons were166 detected. Sequences of G. hodgsoni were unambiguous at 658 bp (no insertions or deletions).167

However, sequences of C. nivicolus and A. monoculata contained ambiguous base pair168 assignments which could not be easily resolved, so sequences were further trimmed at both169 ends, resulting in sequence fragments of 547 bp (181 codons) for C. nivicolus and 527 bp170

(175 codons) for A. monoculata. Two additional C. nivicolus sequences were also obtained171 from GenBank (Accession numbers DQ285403 and DQ285404). 172

Sequences for all species were initially examined in the context of generating a single173 neighbour-joining tree using a Kimura 2-parameter distance model (Kimura, 1980). All174 duplicate sequences were identified and removed to include only unique haplotypes in175 subsequent analyses. Due to the lack of publically available sequence data for taxa that share176 a recent common ancestor with our ingroup taxa (and that did not approach saturation),177 analyses were run unrooted among the ingroup taxa. No significant changes were noted in178 topography between these analyses and ones run previously using Podura aquatic as a test.179

Chi-square tests (X²) as implemented in PAUP* 4.0 (Swofford, 2002) were used to determine180</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s9v1">whether the assumption of equal base frequencies among sites was violated on all sites and181 on third codon positions only. JModel test 2.1.2 (Posada, 2008) was used to determine the182 most appropriate substitution model for Maximum Likelihood (ML) analysis. Settings were183 as follows: 11 substitution schemes (88 models), base frequencies +F, rate variation +I, +Γ,184 set to BioNJ. The model selected for the data set was GTR + I +Γ (-lnL = 1,590.9). Maximum185

Likelihood heuristic searches were conducted using this model in MEGA 5.10 (Tamura et al.,186

2011) using 1000 bootstrap replicates. Maximum Parsimony (MP) analyses were performed187 in PAUP* using 1000 full-heuristic search bootstrap replicates. 188

MrBayes 3.2.6 (Huelsenbeck &amp; Ronquist, 2001) was used to conduct a Bayesian Inference189 analysis. A general time reversal model (GTR +I +Γ) was used, with a log normal relaxed190 clock model and speciation Yule process as the tree prior. The Markov chain Monte Carlo191

(MCMC) was set to 1,100,000 generations, sampling trees every 200. A burn in of 100,000192 trees was determined by plotting log-likelihood values against generation time in TRACER193

(Rambaut &amp; Drummond, 2007) and checking for the point at which normalization occurred.194

The majority rule tree was acquired from the 11,004 trees sampled after the burn in period.195

The tree was then visualized in Tree Annotator (Drummond et al., 2012).  196

Sequences for G. hodgsoni and C. nivicolus were split into separate data sets for analysis in197 the program TCS 1.21 (Clement et al., 2000) and to construct networks of sequence198 haploytpes. Single representatives of each haplotype were used in the final analysis to199 simplify files, and sequences of C. nivicolus were trimmed at 547 bp to avoid anomalies, as200 described above. The A. monoculata sequences were not included in these analyses as they201 were only collected from a single site and consisted of only two similar haplotypes (&lt;0.2%202 divergence). 203</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s10v1">Uncorrected pair-wise genetic distances between COI sequences for populations at different204 locations were also calculated for the G. hodgsoni and C. nivicolus data sets in MEGA 5.10.205

The likelihood ratio test did not detect evidence of significant rate heterogeneity for G.206 hodgsoni (X2=113.06; p&lt;0.001; d.f=14) or C. nivicolus (X2=141.15; p&lt;0.001; d.f=10).207

Approximate geological timing of isolation for the populations was estimated through208 molecular clock analyses in BEAST 1.8.2 (Drummond et al., 2012). Files generated in209

BEAUti used a General Time Reversal model (GTR + I + Γ) with speciation Yule Processes210 as the tree prior and the same MCMC set up as used for the BI tree analysis. A strict clock211 model with a fixed rate of 0.0115 was used to simulate 2.3% sequence divergence per million212 years, as determined using insect mitochondrial data (Brower 1994; Juan et al., 1996; Quek et213 al., 2004; McGaughran et al., 2010). Despite being calibrated for insects, the 2.3% sequence214 divergence per million years was considered a suitable estimate for Collembola as both taxa215 have similar life cycles (McGaughran et al., 2010).  216

Results217

Of the 658 bp analysed for G. hodgsoni, 515 characters were constant, 22 were parsimony218 informative and the remaining 121 were parsimony uninformative. The nucleotide219 composition averaged across all sequences showed an A-T bias of 64.0% (A = 27.7%, T =220

36.7%, C = 19.3%, G = 16.7%). Nucleotide frequencies were not significantly different221 among sequences for all codon positions (X2 = 2.19, p = 1.0, d.f = 48) or for third codon222 positions only (X2 = 7.18, p = 1.0, d.f = 48). Of the 549 bp analysed for C. nivicolus, 433223 characters were constant, 22 were parsimony informative and the remaining 94 were224 parsimony uninformative. The nucleotide composition averaged across all sequences showed225 an A-T bias of 61.4% (A = 25.8%, T = 35.6%, C = 20.4%, G = 18.2%). Base pair frequencies226 for C. nivicolus were not significantly different among sequences for all codon positions (X2227</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s11v1">= 1.41, p = 1.0, d.f = 36) or for third codon positions only (X2 = 5.77, p =1.0, d.f = 36). Of the228

527 bp (175 codons) analysed for A. monoculata, 408 characters were constant, 1 was229 parsimony informative and the remaining 118 were parsimony uninformative. The nucleotide230 composition averaged across all sequences showed an A-T bias of 59.0% (A = 23.9%, T =231

35.1%, C = 22.3%, G = 18.7%). Base pairs were not significantly different among sequences232 for all codon positions (X2 = 3.39, p = 1.0, d.f = 21) or for third codon positions only (X2 =233

11.55, p = 0.95, d.f = 21).  234

Phylogenetic Analysis235

A Maximum Likelihood (ML) tree is shown in Fig. 2. Tree constructions for Maximum236

Parsimony (Fig.3) and Neighbour Joining (data not shown) showed similar topology and237 node support. Linking nodes between the haplotype G16 and the rest of the G. hodgsoni238 haplotypes had 100% bootstrap support in the ML and MP trees. The linking node between239 the C. nivicolus haplotypes at Springtail Point and at Mt Gran also received 100% bootstrap240 support in the ML and MP trees. Bootstrap values for the Mt England C. nivicolus haplotypes241 indicated high support from both the ML and MP trees. The topology for the G. hodgsoni242 haplotypes differed in the ML from both the MP and BI trees. Two clusters were apparent,243 with 0.99 bootstrap support for the node. Collection locations of haplotypes were mixed244 between both clusters. The topology of the BI tree was also similar to all other trees for G.245 hodgsoni, C. nivicolus and A. monoculata. Posterior probability values between C. nivicolus246 haplotypes at Springtail Point and at Mt Gran was 1.00, and also 1.00 between the Mt247

England and Mt Gran group (Fig. 4). The topology and node support of these trees supports248 the presence of high genetic structuring across the Mackay Glacier. 249

Haplotype networks250</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s12v1">The geographic distribution of sequence haplotypes for G. hodgsoni and C. nivicolus was251 investigated using haplotype joining networks. Subsequent haplotype assignments and their252 collection locations are shown in Table 1. Sixteen haplotypes were found from 67 G.253 hodgsoni sequences. Maximum connection steps were fixed at 40 in order to connect254 haplotype G16 to the rest of the haplotypes (Fig. 5). This network revealed 10 1-step255 haplotypes, three 2-step haplotypes, two 3-step haplotypes and one 35-step haplotype. The256 most divergent haplotype shown by this analysis was G16, representing three individuals257 from Mt Gran. This difference was also supported by divergence values and phylogenetic258 trees (Figs 2, 3, 4, 7). The remainder of the network which included haplotypes from the St259

John's Range and Mt Seuss did not show high geographic structure, similar to that observed260 in the tree-based approaches. 261

Twelve haplotypes were found from 22 C. nivicolus sequences. Maximum connection steps262 were fixed at 30 in order to connect the Mt Gran and Mt England haplotypes to the Springtail263 point haplotypes (Fig. 6). This network revealed nine 1-step haplotypes, two 3-step264 haplotypes and one 16-step haplotype. This network analysis showed two groups of265 haplotypes that were connected by 16 missing mutational steps. These two groups266 corresponded to populations at Springtail Point on the south edge of Mackay Glacier, and Mt267

Gran and Mt Seuss to the north and in the centre of the glacier respectively. This difference268 was supported by divergence values and phylogenetic trees. The 2-step link to haplotypes at269

Mt England was also supported by divergence values and phylogenetic trees. 270

COI sequence divergence and molecular clock estimates271

Genetic distances ranged from 0.0-8% for G. hodgsoni and 0.00-4.2% for C. nivicolus272

(Fig.7). Greatest differences were found between haplotype G16 at Mt Gran and the273 remainder of the G. hodgsoni haplotypes, and the genetic distance between C. nivicolus274</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s13v1">haplotypes at Mt Gran and Mt England, and those at Springtail Point. The St John's Range275 and Mt Seuss G. hodgsoni haplotypes showed an average divergence of 0.6% within the276 group (Fig.7). The single haplotype, G16, at Mt Gran showed an average of 7.6% sequence277 divergence from the other haplotypes. 278

The average sequence divergences among C. nivicolus haplotypes within each location were279

0.1% at Mt Gran, 0.2% at Springtail Point and 0.2% at Mt England. Sequence divergences280 between locations showed the haplotypes at Mt Gran to be an average of 4.0% divergent from281 haplotypes at Mt England. Similarly, Springtail Point haplotypes were an average of 3.8%282 divergent from those found at Mt Gran. The Mt Gran and Mt England haplotypes were the283 most similar, with 0.8% sequence divergence between them. 284

Based on a strict molecular clock rate of 2.3% sequence divergence per million years, these285 populations are all likely to have diverged within the last 4 My (Figs 7, 8). The oldest286 estimated divergence dated the genetic separation of G. hodgsoni haplotypes at Mt Gran287

(G16) and those in the St John's Range and at Mt Seuss at 3.8 Mya. Divergence dates288 between the three C. nivicolus populations suggested that the Springtail Point haplotypes289 diverged from the Mt Gran - Mt Seuss population 1.44 Mya. The difference between290 haplotypes from Mt Gran and Mt Seuss relative to those at Mt England is much more recent291 by comparison, estimated at 0.38 Mya. 292

Discussion293

Our mitochondrial DNA (COI) analysis of 97 Antarctic springtails from three taxonomic294 species revealed highly divergent populations across 65 km within the Mackay Glacier.295

Populations of Gomphiocephalus hodgsoni and Cryptopygus nivicolus on the lower slopes of296

Mt Gran were shown to be an average of 7.6% and 3.8% divergent from their nearest297 neighbours. For G. hodgsoni, this represents a considerably greater genetic divergence among298</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s14v1">populations than the 2.4% divergence previously found for this species throughout the299

McMurdo Dry Valleys (Stevens &amp; Hogg, 2003; Nolan et al., 2006; McGaughran et al.,300

2008). High genetic structure, within both putative species, suggests that populations may301 have survived in situ since the Antarctic continent became fully glaciated. Given the302 elevations of surrounding mountains it is possible that several locations such as Mt Gran303

(2235 m) and Mt Seuss (1190 m) protruded above the advancing Mackay Glacier, and304 remained so since the early Pliocene (Janetschek, 1967a; Clapperton &amp; Sugden, 1990). In305 particular, this area is known to contain the highest species diversity of springtails in southern306

Victoria Land, with G. hodgsoni, C. nivicolus and A. monoculata all known from this area307

(Gressitt et al., 1963). The species diversity of mites, lichens and mosses have also been308 shown to be high in the Mackay Glacier region relative to other nearby areas such as the Dry309

Valleys (Demetras et al., 2010; Green et al., 2011). This suggests that this area has served as310 a glacial refuge for multiple taxa during the last 5 Mya.311

We now also highlight the potential for species-level genetic divergences within two312 springtail taxa for populations on opposite sides of the Mackay Glacier, which may indicate313 early stages of speciation. Our data suggest that the population of G. hodgsoni present on the314 lower slopes of Mt Gran has been isolated from other known G. hodgsoni populations since315 the Mid-Pliocene (4 Mya). Similarly, the population of C. nivicolus from the same location316 has been isolated from a neighbouring population at Springtail Point by as much as 1.4 Mya.317

The occurrence of A. monoculata at Springtail Point, coupled with the highly divergent318 populations at Mt Gran supports the notion of high arthropod diversity for this area. 319



The differences in divergence estimates for G. hodgsoni (3.8 Mya) and C. nivicolus (1.4320

Mya) may be the result of different evolutionary histories (e.g. later isolation) or possibly321 differences in mutation rates. For example, Stevens &amp; Hogg (2006) suggested that differing322</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s15v1">mutation rates may exist between G. hodgsoni and the mite Stereotydeus mollis. However,323 little is known about the life history of C. nivicolus. The lack of ecological knowledge for C.324 nivicolus also makes it difficult to predict its dispersal abilities. Dispersal events in Antarctica325 are likely to be rare, and often accidental, making it difficult to attribute the presence of a326 species to ecological gradients (Janetschek, 1967b; Magalhães et al., 2012). G. hodgsoni is327 known to survive floating on both sea and fresh water, and dispersal events through wind or328 accidental carriage by birds is also possible (Stevens &amp; Hogg, 2002; Hawes, 2011;329

McGaughran et al., 2011a, 2011b). 330

As Mackay Glacier is an outlet glacier for the EAIS, it is unlikely to have undergone331 significant retreat during the interglacial periods of the Pleistocene as many of the alpine332 glaciers did (Clapperton &amp; Sugden, 1990; Sugden et al., 1999). This appears to have isolated333 the Mt Gran population of G. hodgsoni from the populations on Mt Seuss in the centre of the334 glacier, and those in the St John's Range bordering Victoria Valley. It is possible that the335 presence of haplotypes from the St John's range in the Mt Seuss population relate to recent336 dispersal since the last glacial maximum. The sharing of C. nivicolus haplotypes between Mt337

Gran and Mt Seuss also indicates potentially recent dispersal from Mt Gran across the338 glacier. Hawes, (2011) suggested that potential dispersal mechanisms may work in concert,339 whereby individuals could be wind-blown onto glaciers and then moved by glacial surface340 streams.  The stochastic nature of dispersal events in Antarctica may explain why G.341 hodgsoni has yet to disperse from the Mt Gran population. 342

One species, A. monoculata, was found at only one location in our study area, although343 another isolated population is known from Mt Murray 150 km to the north (Gressitt et al.,344

1963). Similarly, haplotypes of C. nivicolus present at this site were not found elsewhere in345 our study area. Springtail Point is in an 'up-glacier' position, making dispersal through346</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s16v1">temporary melt water to more sea-ward locations possible. However, there was no evidence347 of water courses being formed by temporary streams in this area, and visual assessment of348 snow banks that surround the site indicate they have changed little since a previous visit349

(Gressitt et al., 1963). Even with surface water, the dispersal mechanisms used by other350 springtail species such as wind and stream flow may be limited for A. monoculata. The loss351 of pigmentation, limited tolerance of UV light and presence deeper in the soil profile352

(Janetschek, 1967a) make it less likely that A. monoculata would experience accidental353 dispersal by water or wind movement. 354

We conclude that the Mackay Glacier has provided a sufficient dispersal barrier to promote355 and maintain high levels of genetic divergence in two Antarctic springtail species endemic to356 southern Victoria Land. This isolation likely occurred around the early Pliocene (4 Mya), and357 has been maintained by on-going glaciations during the Pleistocene. The high genetic358 diversity, both at the population and species level, suggests that high altitude sites in this359 region have served as glacial refugia over the past 4 Mya. The isolation of these sites360 highlights the potential for high genetic diversity to be maintained on a small scale among the361 fragmented ice-free areas of Antarctica. Accordingly, we suggest that conservation efforts be362 directed toward maintaining and protecting the integrity of highly fragmented landscapes363 within the Transantarctic Mountains of the Ross Dependency. 364

365</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s17v1">Acknowledgements366

We thank M. Knox, U. Nielsen, D. Wall, and D. McKnight (US Antarctic LTER Programme)367 for helpful advice and/or assistance in the field/lab, and members of the Pacific368

Biosystematics Research group (University of Waikato) for discussion and comments during369 manuscript preparation. Glen Stichbury provided help in preparing Fig. 1 which was derived370 from the SCAR Antarctic Digital Database and the Landsat Image Mosaic of Antarctica371

(LIMA) project. Financial support was provided to KRB through a New Zealand Post372 scholarship administered by Antarctica New Zealand, a University of Waikato Masters373

Research Scholarship, and The International Centre for Terrestrial Antarctic Research374

(ICTAR) Young Investigator Award. Field work was supported by Antarctica New Zealand375 and the US National Science Foundation through the McMurdo LTER NSF OPP grant376

1115245. Sequencing at the Biodiversity Institute of Ontario was supported by funding from377 the Government of Canada through Genome Canada and the Ontario Genomics Institute.378



379



380



381



382



383



384



385



386



387



388



389</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s18v1">Literature Cited390

Adams B, Bardgett R., Ayres E, Wall DH, Aislabie J, Bamforth S, Bargagli R, Cary C,391

Cavacini P, Connell L. 2006. Diversity and distribution of Victoria Land biota. Soil392

Biology and Biochemistry 38: 3003–3018. 393

Adhikari BN, Wall DH, Adams BJ. 2010. Effect of slow desiccation and freezing on gene394 transcription and stress survival of an Antarctic nematode. Journal Of Experimental395

Biology 213: 1803–1812.396

Brower A. 1994. Rapid morphological radiation and convergence among races of the397 butterfly Heliconius erato inferred from patterns of mitochondrial DNA evolution.398

Proceedings of the National Academy of Sciences of the United States of America 91:399

6491–6495. 400

Clapperton CM, Sugden D. 1990. Late Cenozoic glacial history of the Ross Embayment,401

Antarctica. Quaternary Science Reviews 9: 253–272. 402

Clement M, Posada D, Crandall K. 2000. TCS: a computer program to estimate gene403 genealogies. Molecular Ecology 9: 1657–1660. 404

Convey P, Stevens MI, Hodgson DA, Smellie JL, Hillenbrand CD, Barnes DKA, Clarke A,405

Pugh PJA, Linse K, Cary SC. 2009. Exploring biological constraints on the glacial406 history of Antarctica. Quaternary Science Reviews 28: 3035–3048. 407

Demetras NJ, Hogg ID, Banks JC, Adams BJ. 2010. Latitudinal distribution and408 mitochondrial DNA (COI) variability of Stereotydeus spp. (Acari: Prostigmata) in409

Victoria Land and the central Transantarctic Mountains. Antarctic Science 22: 749–756. 410</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s19v1">Drummond AJ, Suchard MA, Xie D, Rambaut A. 2012. Bayesian Phylogenetics with411

BEAUti and the BEAST 1.7. Molecular Biology and Evolution, 29, 1969–1973. 412

Folmer O, Black M, Hoeh W, Lutz R, Vrijenhoek R. 1994. DNA primers for amplification of413 mitochondrial cytochrome c oxidase subunit I from diverse metazoan invertebrates.414

Molecular Marine Biology and Biotechnology 3: 294–9. 415

Frati F, Spinsanti G, Dallai R. 2001. Genetic variation of mtCOII gene sequences in the416 collembolan Isotoma klovstadi from Victoria Land, Antarctica: evidence for population417 differentiation. Polar Biology 24: 934–940. 418

Green TGA, Sancho LG, Türk R, Seppelt RD, Hogg ID. 2011. High diversity of lichens at419

84°S, Queen Maud Mountains, suggests preglacial survival of species in the Ross Sea420 region, Antarctica. Polar Biology, 34, 1211–1220. 421

Greenslade P. 2015. Synonymy of two monobasic Anurophorinae genera (Collembola:422

Isotomidae) from the Antartctic Continent. New Zealand Entomologist 38: 134-141.423

Gressitt J, Leech R, Wise KAJ. 1963. Entomological investigations in Antarctica. Pacific424

Insects 5: 287–304. 425

Gressitt J. 1967. The Fauna. Terrestrial life of Antarctica (ed. by V. Bushnell), pp. 17-24. 426

Antarctic map folio series, New York. 427

Hajibabaei M, Janzen DH, Burns JM, Hallwachs W, Hebert PDN. 2006. DNA barcodes428 distinguish species of tropical Lepidoptera. Proceedings of the National Academy of429

Sciences of the United States of America 103: 968–71. 430</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s20v1">Hawes TC. 2011. Rafting in the Antarctic springtail, Gomphiocephalus hodgsoni. Antarctic431

Science 23: 456–460. 432

Hawes TC, Torricelli G, Stevens MI. 2010. Haplotype diversity in the Antarctic springtail433

Gressittacantha terranova at fine spatial scales-a Holocene twist to a Pliocene tale.434

Antarctic Science 22: 766–773.435

Hogg ID, Wall DH. 2012. Extreme Habitats: Polar Deserts. Life at Extremes: Environments,436

Organisms and Strategies for Survival (ed. by E. Bell). CAB International, Cambridge,437

UK. 438

Hogg ID, Cary CS, Convey P, Newsham KK, O’Donnell AG, Adams BJ, Aislabie J, Frati F,439

Stevens MI, Wall DH. 2006. Biotic interactions in Antarctic terrestrial ecosystems: Are440 they a factor? Soil Biology and Biochemistry 38: 3035–3040. 441

Hogg ID, Stevens MI. 2002. Soil Fauna of Antarctic Coastal Landscapes. Ecological Studies442

154: 265–282. 443

Huelsenbeck J, Ronquist F. 2001. MRBAYES: Bayesian Inference of Phylogeny.444

Bioinformatics 17: 754–755. 445

Ivanova NV, Deewaard JR, Hebert PDN. 2006. An inexpensive, automation-friendly protocol446 for recovering high-quality DNA. Molecular Ecology Notes 6: 998-1002.447

Janetschek H. 1967a. Arthropod ecology of south Victoria Land. Antarctic Research Series448

10: 205–293. 449</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s21v1">Janetschek H. 1967b. Growth and maturity of the springtail Gomphiocephalus hodgsoni450

Carpenter, from south Victoria Land and Ross Island. Antarctic Research Series 10: 295451

–305. 452

Juan C, Oromi P, Hewitt GM. 1996. Phylogeny of the genus Hegeter (Tenebrionidae,453

Coleoptera) and its colonization of the Canary Islands deduced from Cytochrome454

Oxidase I mitochondrial DNA sequences. Heredity 76: 392–403. 455

Kimura M. 1980. A simple method for estimating evolutionary rate of base substitutions456 through comparative studies of nucleotide sequences. Journal of Molecular Evolution457

16: 111-120.458

Lewis A, Marchant D, Ashworth A, Hemming S, Machlus M. 2007. Major middle Miocene459 global climate change: Evidence from East Antarctica and the Transantarctic Mountains.460

Geological Society of America Bulletin 119: 1449-1461.461

Levy J. 2012. How big are the McMurdo Dry Valleys? Estimating ice-free area using Landsat462 image data. Antarctic Science 25: 119-120.463

Magalhães C, Stevens MI, Cary SC, Ball BA, Storey BC, Wall DH, Türk R, Ruprecht U.464

2012. At limits of life: multidisciplinary insights reveal environmental constraints on465 biotic diversity in continental Antarctica. PloS one 7: e44578. 466

McGaughran A, Hogg ID, Convey P. 2011a. Extended ecophysiological analysis of467

Gomphiocephalus hodgsoni (Collembola): flexibility in life history strategy and468 population response. Polar Biology, 34, 1713–1725. 469

McGaughran A, Stevens MI, Hogg ID, Carapelli A. 2011b. Extreme Glacial Legacies: A470

Synthesis of the Antarctic Springtail Phylogeographic Record. Insects 2: 62–82. 471</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s22v1">McGaughran A, Hogg ID, Stevens MI. 2008. Patterns of population genetic structure for472 springtails and mites in southern Victoria Land, Antarctica. Molecular Phylogenetics473 and Evolution 46: 606–18. 474

McGaughran A, Torricelli G, Carapelli A, Frati F, Stevens MI, Convey P, Hogg ID. 2010.475

Contrasting phylogeographical patterns for springtails reflect different evolutionary476 histories between the Antarctic Peninsula and continental Antarctica. Journal of477

Biogeography 37: 103–119. 478

Nkem J, Wall D, Virginia R., Barrett JE, Broos E, Porazinska DL, Adams B. 2006. Wind479 dispersal of soil invertebrates in the McMurdo Dry Valleys, Antarctica. Polar Biology480

29: 346–352. 481

Nolan L, Hogg ID, Stevens MI, Haase M. 2006. Fine scale distribution of mtDNA haplotypes482 for the springtail Gomphiocephalus hodgsoni (Collembola) corresponds to an ancient483 shoreline in Taylor Valley, continental Antarctica. Polar Biology 29: 813–819. 484

Posada D. 2008. jModelTest: phylogenetic model averaging. Molecular Biology and485

Evolution 25: 1253–6. 486

Quek S, Davies S, Ilino T, Pierce N. 2004. Codiversification in an ant-plant mutualism: stem487 testure and the evolution of host use in Crematogaster (Formicidae: Myrmicinae)488 inhabitants of Macaranga (Euphorbiaceae). Evolution 58: 554–570. 489

Rambaut A, Drummond A. 2007. Tracer v1.4, Available from490 http://beast.bio.ed.ac.uk/Tracer.491

Salmon JT. 1965. An index to the Collemnola. Bulletin, Royal Society of New Zealand 7.492</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s23v1">Sinclair BJ, Stevens MI. 2006. Terrestrial microarthropods of Victoria Land and Queen Maud493

Mountains, Antarctica: Implications of climate change. Soil Biology and Biochemistry494

38: 3158–3170. 495

Stevens MI, D’Haese C. 2014. Islands in ice: isolated populations of Cryptopygus sverdrupi496

(Collembola) among nunataks in the Sor Rondane Mountains, Dronning Maud Land,497

Antarctica Biodiversity 15: 169-177.498

Stevens MI, Hogg ID. 2002. Expanded distributional records of Collembola and Acari in499

Southern Victoria Land, Antarctica. Pedobiologia 46: 485-495.500

Stevens MI, Hogg ID. 2003. Long-term isolation and recent range expansion from glacial501 refugia revealed for the endemic springtail Gomphiocephalus hodgsoni from Victoria502

Land, Antarctica. Molecular Ecology 12: 2357–2369. 503

Stevens MI, Hogg ID. 2006. Contrasting levels of mitochondrial DNA variability between504 mites (Penthalodidae) and springtails (Hypogastruridae) from the Trans-Antarctic505

Mountains suggest long-term effects of glaciation and life history on substitution rates,506 and speciation processes. Soil Biology and Biochemistry 38: 3171–3180. 507

Stevens MI, Greenslade P, Hogg ID, Sunnucks P. 2006. Southern hemisphere springtails:508 could any have survived glaciation in Antarctica? Molecular Biology and Evolution 23:509

874-882.510

Sugden DE, Summerfield MA, Denton GH, Wilch TI, McIntosh WC, Marchant DR., Rutford511

RH. 1999. Landscape development in the Royal Society Range, southern Victoria Land,512

Antarctica: stability since the mid-Miocene. Geomorphology 28: 181–200. 513</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s24v1">Suggate RP. 1990. Late Pliocene and Quaternary glaciations of New Zealand. Quaternary514

Science Reviews 9: 175–197. 515

Swofford D. 2002. PAUP*: Phylogenetic Analysis Using Parsimony, version 4.0b10 for516

Macintosh. 517

Tamura K, Peterson D, Peterson N, Stecher G, Nei M, Kumar S. 2011. MEGA5: Molecular518

Evolutionary Genetics Analysis using Maximum Likelihood, Evolutionary Distance, and519

Maximum Parsimony Methods. Molecular Biology and Evolution 28: 2731-2739.520

Wise KAJ. 1971. The Collembola of antarctica. Pacific Insects Monograph 25: 57–74.521



522

Biosketch523

KRB is an MSc graduate from the University of Waikato with interests in animal524 conservation and connectivity among natural populations. Her primary target taxa are525 terrestrial Collembola and freshwater macroinvertebrates. She is also interested in the526 evolution of the New Zealand and Antarctic landscapes.527

Author contributions:528

IDH, KRB, BJA and PDNH conceived of the research and obtained funding. KRB and IDH529 conducted the field work and KRB conducted the primary analyses and was lead author of530 the manuscript in conjunction with IDH BJA and PDNH. All authors reviewed and531 contributed revisions to the final version of the manuscript.532





533</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s25v1">Table 1: Haplotypes, collection locations, coordinates and sequences (BOLD Process Id) associated with each haplotype for three species of Antarctic springtail. Two Mt England

C. nivicolus sequences (N11, N12) were retrieved from GenBank. 

Haplotype # Location Co-ordinates (south – east) Process Id's

G. hodgsoni

 





G1

St John's Range -77.280   161.731

  ANTSP131  ANTSP134  ANTSP136 ANTSP137  ANTSP138  ANTSP140  ANTSP141  ANTSP143  ANTSP129  ANTSP193  ANTSP151

G2







ANTSP133 ANTSP135 ANTSP139 ANTSP211 ANTSP212 ANTSP132

G3 



-77.208   161.700 ANTSP213 ANTSP215

G4 



-77.285   161.726 ANTSP150

G5 





ANTSP142

G6 





ANTSP146

G7 



-77.208   161.700 ANTSP209

G8 





ANTSP210

G9 





ANTSP216

G10 



-77.285   161.726 ANTSP217

G11  

-77.280   161.731 ANTSP144 ANTSP145 ANTSP147 ANTSP148 ANTSP149 ANTSP191 ANTSP192 ANTSP207 ANTSP219 ANTSP214 ANTSP128 ANTSP218Mt Seuss

-77.034   161.731

G12





ANTSP154 ANTSP157  ANTSP158  ANTSP159 ANTSP160 ANTSP163  ANTSP164  ANTSP165  ANTSP168  ANTSP169  ANTSP172 ANTSP174  ANTSP175  ANTSP220  ANTSP222  ANTSP221  ANTSP223  ANTSP224  ANTSP152  ANTSP225

G13 





ANTSP162 ANTSP173</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s26v1">G14 





ANTSP166 ANTSP153 ANTSP167

G15



77.034   161.731 ANTSP161

G16  Mt Gran -76.966   161.179 ANTSP201 ANTSP202 ANTSP200

C. nivicolus 



 





N1

Springtail Point -77.167   160.710  ANTSP121 ANTSP188 ANTSP190 ANTSP230 ANTSP119

N2





ANTSP2234 ANTSP228

N3





ANTSP231 ANTSP226

N4 





ANTSP227

N5 





ANTSP118

N6  Mt Gran -76.966   161.179 ANTSP233

N7 





ANTSP199 ANTSP197

N8  



ANTSP156 ANTSP124

Mt Seuss

-77.034   161.731

N9 





ANTSP155

N10 





ANTSP170

N11 M England -77.046   162.450 DQ285403

N12





DQ285404

A. monoculata 



 





A1

Springtail Point -77.168   160.710 ANTSP196 ANTSP235

A2





ANTSP204 ANTSP205 ANTSP194 ANTSP195 ANTSP203



534</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s27v1">535

List of Figures536

Figure 1: Sampling sites and Collembola species’ locations in the Mackay Glacier vicinity. Two C.537 nivicolus specimens were taken from GenBank and were collected from Mt England in 2005. Map538 adapted from the SCAR Antarctic Digital Database and the Landsat Image Mosaic of Antarctica539

(LIMA) project.540

Figure 2: Maximum Likelihood phylogram constructed in MEGA 5.10, based on the GTR+I+Γ model541 derived from jModelTest, using 97 individual COI sequences reduced to unique haplotypes.Bootstrap542 values greater than 50 are shown. Tree is drawn to scale and branch lengths are the number of543 substitutions per site. Collection locations are indicated for genetically distinct groups. 544

Figure 3: Maximum Parsimony Phylogram constructed in PAUP*, using 97 individual COI sequences545 reduced to unique haplotypes.. Bootstrap values greater than 50 are shown. Tree is drawn to scale and546 branch lengths are the number of changes over the whole sequence. Collection locations are indicated547 for genetically distinct groups. 548

Figure 4: Bayesian Inference Phylogram constructed in MrBayes 3.2.6 based on the GTR+I+Γ model549 derived from jModelTest, using 97 individual COI sequences reduced to unique haplotypes. Posterior550 probabilities for haplotype group nodes are presented above 0.5. Tree is drawn to scale and branch551 lengths are measured in the number of changes per site. Collection locations are indicated for552 genetically distinct groups. 553

Figure 5: Haplotype network analysis for 16 haplotypes from 67 individuals of G. hodgsoni.554

Haplotypes are indicated by their codes as referred to in Table 1. Missing haplotypes or mutational555 steps are indicated by black dots, or are collapsed into a count of missing steps as in the single white556 square. 557</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s28v1">Figure 6: Haplotype network analysis for 12 haplotypes from 22 individuals of C. nivicolus.558

Haplotypes are indicated by their codes as referred to in Table 1. Missing haplotypes or mutational559 steps are indicated by black dots, or are collapsed into a count of missing steps as in the white square. 560

Figure 7: Genetic distances based on mitochondrial COI sequences of 97 springtails covering 30561 unique haplotypes. Haplotype codes refer to those in Table 1. Collection locations for each haplotype562 are indicated in the bar at the top and side of the table.563

Figure 8: Estimated divergence times for populations of G. hodgsoni (circle) and C. nivicolus564

(squares). The timeline on the left is in millions of years. Overarching geologic events are presented565 in the appropriate time zones. Each bar indicates the divergence range between populations as566 indicated by the associated number pair. Each number refers to haplotypes from geographic locations567 as follows: 1 = G. hodgsoni haplotypes from the St John's range and Mt Seuss; 2 = the G. hodgsoni568 haplotype at Mt Gran; 3 = C. nivicolus haplotypes from Springtail Point; 4 = C. nivicolus haplotypes569 fromMt Gran and Mt Seuss; 5 = C. nivicolus haplotypes fromMt England. 570



571



572



573



574



575



576



577



578



579</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s29v1">580



581



582

Fig. 1583</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s30v1">584



585

Fig. 2586</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s31v1">587



588



589



590

Fig. 3591



592



593</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s32v1">594



595



596



597



598



599



600



601



602



603



604



605



606



607



608



609



610



611



612



613



614



615

Fig. 4616



617



618</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s33v1">619



620



621



622



623



624

Fig. 5625



626



627



628



629



630</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s34v1">631



632



633



634



635



636

Fig. 6637



638



639



640



641



642</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s35v1">643

Fig. 7644</pre>
        </div>
      </section>
      <section data-type="">
        <div class="content">
          <pre id="s36v1">645

Fig. 8646</pre>
        </div>
      </section>
    </article>
  </body>
</html>
